avogadro/io/pdbformat.cpp

```diff
@@ -132,7 +159,7 @@ bool PdbFormat::read(std::istream& in, Core::Molecule& mol)
       }
     }
   } // End while loop
-  mol.perceiveBondsSimple();
+  // mol.perceiveBondsSimple();
```

Comment:
You still need to do this, although CONECT records help, you still want to bond HETATM. The main thing would be to ensure that perceiveBondsSimple() ignores the bonds and bond orders that you've added already.

Reply:
Basically, our experience in the Open Babel project (and others) is that CONECT records can't always be trusted, so still run the perceiveBondsSimple at the end here.